specs/core/1_shard-data-chains.md

```diff
@@ -282,9 +288,9 @@ Let:
 * `candidate` be a candidate `ShardBlock` for which validity is to be determined by running `is_valid_shard_block`
 
 ```python
-def is_valid_shard_block(beacon_blocks: List[BeaconBlock],
+def is_valid_shard_block(beacon_blocks: TypingList[BeaconBlock],
                          beacon_state: BeaconState,
-                         valid_shard_blocks: List[ShardBlock],
+                         valid_shard_blocks: Iterable[ShardBlock],
```

Comment:
Would `valid_shard_blocks` be `Tuple/List`? If so, recommend using `Sequence`. 
```suggestion
                         valid_shard_blocks: Sequence[ShardBlock],
```

Reply:
Put sequece in more places now, thanks for suggestion.